### PR TITLE
convert docs/format.rst to tags(5) man page

### PR DESCRIPTION
**Pull request by masatake**

### CODE REVIEW

**Comment by masatake** [comment]
I will merge this before merging #2199.
The documentation for #2199 may need this change.

File: docs/output-format.rst [hunk 1]
```diff
@@ -13,6 +13,9 @@ This section deals with individual output-format topics.
 
 ``--output-format=`` can be used for choosing an output format.
 
+About the output format specified with ``e-ctags`` and ``u-ctags``,
+see format(5) man page.
```
Comment (k-takata) [inline docs/output-format.rst]: Should be "tags(5)"?

File: man/Makefile [hunk 2]
```diff
@@ -39,14 +39,19 @@ CONFIGURE_FLAGS =
...
+IN_SOURCE_FILES = \
 	ctags.1.rst.in \
 	ctags-optlib.7.rst.in \
 	ctags-incompatibilities.7.rst.in \
 	\
 	$(NULL)
 
-rst_files  = $(basename $(SOURCE_FILES))
+SOURCE_FILES = \
+	format.5.rst \
```
Comment (k-takata) [inline man/Makefile]: Should be "tags.5.rst"?

File: docs/contributions.rst [hunk 3]
```diff
@@ -207,7 +207,7 @@ following two areas.
 Tag file compatibility with Exuberant-ctags
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 We will not accept a patch that breaks the tags file format described
-in "Proposal for extended Vi tags file format" a.k.a. FORMAT file.
+format(5) man page.
```
Comment (k-takata) [inline docs/contributions.rst]: Should be "tags(5)"?

File: Makefile.am [hunk 4]
```diff
@@ -179,15 +179,25 @@ include $(top_srcdir)/makefiles/testing.mak
 include $(top_srcdir)/makefiles/help.mak
 
 if HAVE_RST2MAN
-man_MANS = man/ctags.1 man/ctags-incompatibilities.7 man/ctags-optlib.7
+man_MANS = \
+	man/ctags.1 \
+	\
+	man/format.5 \
```
Comment (k-takata) [inline Makefile.am]: Should be "man/tags.5"?

Comment (masatake) [reply Makefile.am]: !! Thank you.

File: docs/contributions.rst [hunk 5]
```diff
@@ -207,7 +207,7 @@ following two areas.
 Tag file compatibility with Exuberant-ctags
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 We will not accept a patch that breaks the tags file format described
-in "Proposal for extended Vi tags file format" a.k.a. FORMAT file.
+tags(5) man page.
```
Comment (k-takata) [inline docs/contributions.rst]: `s/tags(5)/in tags(5)/` ?

File: docs/output-format.rst (hunk 1 above)
Comment (k-takata) [reply docs/output-format.rst]: Are you missing this?

File: docs/contributions.rst (hunk 5 above)
Comment (masatake) [reply docs/contributions.rst]: Thank you.

**Comment by masatake** [comment]
Squashed.

**Pull request merged**